client/ipa-join.c

```diff
@@ -1064,3 +1521,6 @@ main(int argc, const char **argv) {
 
     exit(ret);
 }
+
+#undef CURL_SETOPT
+#undef ASPRINTF
```

Comment:
Just curious, why undefine them?

Reply:
Just cleaning up after my self ;) Thought it might be a good idea...
But I suppose since it's not a header file, the macro scope is limited to this file. I can remove them if you want me to.

Reply:
Yeah, I don't think they are necessary. Please remove them.